WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -281,7 +281,7 @@ final class OrderListViewModel {
         }
     }
 
-    func displayIPPFeedbackBannerIfEligible() {
+    func displayIPPFeedbackBannerIfEligible() -> SurveyViewController.Source? {
```

Comment:
Now that it returns something, the name should probably change.

The name of `displayIPPFeedbackBannerIfEligible` doesn't suggest that it's going to return a survey source, more that it's going to display the banner. 

Perhaps renaming to `feedbackBannerSurveySource` or something would be an improvement here?

Reply:
Renamed on 77e96f3c6f6c95438485826fa6fa92a4dccb5f2f